tests/testthat/test-shinytest2-tm_a_pca.R

```diff
@@ -0,0 +1,133 @@
+testthat::test_that("e2e - tm_a_pca: Data selection (data_extract) changes eigenvector table", {
+  skip_if_too_deep(5)
+
+  app_driver <- app_tm_a_pca()
```

Comment:
`app_tm_a_pca` is not renamed to `app_driver_tm_a_pca`.

Reply:
:facepalm: